src/js/controller.js

```diff
@@ -445,8 +445,8 @@ export const MediaUIRequestHandlers = {
     if (streamType == StreamTypes.LIVE && autoSeekToLive) {
       MediaUIRequestHandlers['MEDIA_SEEK_TO_LIVE_REQUEST'](media);
     }
-
-    media.play().catch(() => {});
+    // Not all media's play() returns a Promise, e.g. castable-video
+    media.play()?.catch(() => {});
```

Comment:
ah we should probably fix that but this is def the right solution for now. Nice find!